ete3/tools/ete_build_lib/task/iqtree.py

```diff
@@ -29,8 +29,11 @@ def __init__(self, nodeid, alg_phylip_file, constrain_id, model,
             base_args["-st"] = "AA" if seqtype == "aa" else "DNA"            
         else:
             if conf[confname]["-st"].startswith("CODON"):
-                if seqtype == "aa" or "aa" not in GLOBALS["seqtypes"]:
-                    raise ConfigError("IQTREE CODON models require a codon alignmen.\nProvide nucleotide sequences with '-n' and set '--nt-switch-thr 0.0' to ensure codon alignments.")
+                #print(GLOBALS["seqtypes"])
+                #print(seqtype)
+                #if seqtype == "aa" or "aa" not in GLOBALS["seqtypes"]: #logic confusion Ziqi
```

Comment:
could you explain what was the issue here? did you check that your version works as expected?